schema-manager/src/main.rs

```diff
@@ -0,0 +1,273 @@
+// Copyright 2019 Joyent, Inc.
+use std::collections::HashMap;
+use std::fs;
+use std::io::{Error, ErrorKind};
+use std::net::TcpStream;
+use std::process;
+
+use clap::{Arg, App, crate_version, ArgMatches};
+use cmd_lib::{run_fun, FunResult};
+use diesel::connection::SimpleConnection;
+use diesel::prelude::*;
+use rust_fast::client as fast_client;
+use rust_fast::protocol::{FastMessage, FastMessageId};
+use sapi::{SAPI, ZoneConfig};
+use serde::Serialize;
+use serde_json::{Value, json};
+use slog::{error, info, o, Drain, Logger};
+use std::sync::Mutex;
+use string_template::Template;
+use utils::config;
+
+/* TODO
+    - populate service name per shard
```

Comment:
Isn't this already done?

Reply:
PR for manta_setup_boray_schemas function is here: https://github.com/joyent/manta-scripts/pull/6

As for service name population, I may be confused. I thought we had discussed something being inserted into the new tables, is that not the case?